README.md

```diff
@@ -23,7 +22,7 @@ code a _builder_ for compiling Base16 _schemes_ and _templates_.
 
 ### Scheme
 
-A scheme is a list of 16 colors to be used as a base to build themes. To make
+A scheme is a list of 16 colors to be used as a base to build application specific support files. To make
```

Comment:
These are the parts begging for good wording.

Reply:
mb
```suggestion
A scheme is a list of 16 colors to be used as a base to build application specific config files. To make
```

Reply:
I purposely tried to stay away from "config" wording.

Reply:
What about just "application specific files"?